Source/SwiftLintFramework/Rules/ColonRule.swift

```diff
@@ -161,3 +209,89 @@ public struct ColonRule: CorrectableRule, ConfigurationProviderRule {
         }
     }
 }
+
+// MARK: Dictionary Colon Rule
+extension ColonRule {
+
+    public func validateFile(_ file: File, kind: SwiftExpressionKind,
```

Comment:
yeah, mixing `ASTRule` with regular `Rule` is weird ATM. I even thought about making this a new rule.